src/modules/AutoinstSoftware.rb

```diff
@@ -943,6 +945,9 @@ def pmInit
     # @param list calculated post packages
     # @return [void]
     def addPostPackages(calcpost)
+      # filter out already installed packages
+      calcpost.select!{|p| !PackageSystem.Installed(p)}
```

Comment:
You could use `#reject!` instead of `#select!`:

```ruby
calcpost.reject! { |p| PackageSystem.Installed(p) }
```

Anyway, it's a matter of what's more readable for you.